### PR TITLE
Features/7733

### PR DESCRIPTION
**Pull request by TiagoGraf89**
Adds a template script for the invitation email
Fixes the email validation logo

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_250)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_250)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_250)

File: Apps/Database/src/Migrations/20191203012313_UpdateEmailTemplate.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.UpdateData(
+                schema: "gateway",
+                table: "EmailTemplate",
+                keyColumn: "EmailTemplateId",
+                keyValue: new Guid("040c2ec3-d6c0-4199-9e4b-ebe6da48d52a"),
+                column: "Body",
+                value: @"<!doctype html>
+<html lang=""en"">
```
Comment (FuriousLlama) [inline Apps/Database/src/Migrations/20191203012313_UpdateEmailTemplate.cs]: instead of putting these here, can we read them as files?

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_250)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_250)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_250)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_250)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_250)

**Pull request merged**